boilerplate/app/i18n/hi.ts

```diff
@@ -0,0 +1,131 @@
+import demoHi from "./demo-hi"
+import { Translations } from "./en"
+
+const hi: Translations = {
+  common: {
+    ok: "ठीक है!",
+    cancel: "रद्द करें",
+    back: "वापस",
+    logOut: "लॉग आउट", // @demo remove-current-line
+  },
+  welcomeScreen: {
+    postscript:
+      "पीएसटी - शायद आपका ऐप ऐसा नहीं दिखता है। (जब तक कि आपके डिजाइनर ने आपको ये स्क्रीन नहीं दी हों, और उस स्थिति में, इसे लॉन्च करें!)",
+    readyForLaunch: "आपका ऐप, लगभग लॉन्च के लिए तैयार है!",
+    exciting: "(ओह, यह रोमांचक है!)",
+    letsGo: "चलो चलते हैं!", // @demo remove-current-line
+  },
+  errorScreen: {
+    title: "कुछ गलत हो गया!",
+    friendlySubtitle:
+      "यह वह स्क्रीन है जो आपके उपयोगकर्ता प्रोडक्शन में देखेंगे जब कोई त्रुटि होगी। आप इस संदेश को कस्टमाइज़ करना चाहेंगे (जो `app/i18n/hi.ts` में स्थित है) और शायद लेआउट भी (`app/screens/ErrorScreen`). यदि आप इसे पूरी तरह से हटाना चाहते हैं, तो `app/app.tsx` में <ErrorBoundary> कंपोनेंट की जांच करें।",
```

Comment:
Wouldn't "Sanchalan" or "sanchalta" be a better translation than transliterating "production"? Similarly for customise, "badalna chahenge" would fit better than "customise karna chahenge". I'm using [IAST](https://en.wikipedia.org/wiki/International_Alphabet_of_Sanskrit_Transliteration) here as I don't have a hindi keyboard.

Reply:
Valid point. I've updated the translation.